.eslintrc.js

```diff
@@ -8,4 +10,22 @@ module.exports = {
         ecmaVersion: 2018,
         sourceType: 'module',
     },
+    rules: {
```

Comment:
Should we add rules for not allowing undefined variables and calling global object properties as functions?

Reply:
Got ya covered. They're included in `eslint:recommended`. Check this out: https://eslint.org/docs/rules